http-client/src/test/java/no/nav/familie/http/sts/StsRestClientTest.kt

```diff
@@ -0,0 +1,68 @@
+package no.nav.familie.http.sts
+
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock.*
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.net.URI
+
+class StsRestClientTest {
```

Comment:
Bra test!

Reply:
Tips hvis det blir mange tester for wiremockserver og kjøringen bruker lang tid for å ta ned/opp Wiremockserver: 

Start/Stopp kan trekkes ut i beforeAll/afterAll i companion object og i afterEach kan restartAll ligge

Reply:
Jupp. Selve wiremock er ikke så treg når det ikke er noe springcontext å dra opp. Ellers så hadde jeg nok gjort det sånn.